docs/deployment/nio/direct.md

```diff
@@ -0,0 +1,60 @@
+# Direct Deployment
+
+A Direct Deployment is an instant deployment to instances that **are** externally accessible. 
+
+> **[info] Compatibility**
+>
+> If you can load your instance in the System Designer, you can do a Direct Deployment to it.
```

Comment:
`If you can load your instance in the System Designer, you can deploy directly to it.` ?